cypress/e2e/clue/branch/student_tests/data_card_tool_spec.js

```diff
@@ -106,19 +106,19 @@ context('Data Card Tool Tile', function () {
     it("can create a graph from the data", () => {
       dc.getLinkGraphButton().should('not.be.disabled').click();
       dc.getLinkGraphModalCreateNewButton().click();
-      xyplot.getTile().should("exist").contains("X-Y Plot 1");
-      xyplot.getXYPlotTitle().should("contain", "X-Y Plot 1");
+      xyplot.getTile().should("exist").contains("Data Card Collection 1");
```

Comment:
The new functionality of linking Data Card or Table tiles to an XY plot is such that the XY Plot adopts the the linked tile's title, so I changed this test so it could pass.